src/bundler/bundle_v2.zig

```diff
@@ -1478,8 +1479,8 @@ pub const BundleV2 = struct {
             bundler.options.emit_dce_annotations = config.emit_dce_annotations orelse !config.minify.whitespace;
             bundler.options.ignore_dce_annotations = config.ignore_dce_annotations;
             bundler.options.experimental_css = config.experimental_css;
-            bundler.options.banner = config.banner.toOwnedSlice();
-            bundler.options.footer = config.footer.toOwnedSlice();
+            bundler.options.banner = config.banner.toOwnedSliceLeaky();
+            bundler.options.footer = config.footer.toOwnedSliceLeaky();
```

Comment:
whats the reasoning here? i forget what leaky does (reviewing on phone) but it sounds worse. what frees this memory

Reply:
previously, it leaked

now it doesn't

Reply:
after reading surrounding code i see why this works but it is extremely error-prone for memory management

Reply:
oh i see